### PR TITLE
Example heroku

### PR DESCRIPTION
**Pull request by Danwhy**
Adds more examples, and server for running them on heroku.
ref #3, #4

### CODE REVIEW

File: cypress/integration/test.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ describe("Testing Example App", function() {
   context("Initialising", function() {
     beforeEach(function() {
       cy.visit("http://localhost:8000/examples/index.html");
+      cy.get('[href="#Simple"]').click();
```
Comment (nelsonic) [inline cypress/integration/test.js]: @Danwhy the Cypress tests _appear_ to only cover the `#Simple` use-case.
Am I _missing_ the _range_ tests...?

Comment (Danwhy) [reply cypress/integration/test.js]: No, you're right; I should add some tests in to cover the range functionality. That example didn't exist when I wrote the tests, and I've yet to update them

Comment (nelsonic) [reply cypress/integration/test.js]: @Danwhy feel free to create a `new` issue for `DRP` tests 
(_as they aren't required for this Heroku PR to "pass" it's original acceptance criteria_) 

But IMO, the `DRP` is the most _useful_ use-case 
which is why AirBnB shows them _first_ in their example: http://airbnb.io/react-dates 😉

File: package.json [hunk 2]
```diff
@@ -22,5 +27,9 @@
   "homepage": "https://github.com/dwyl/elm-datepicker#readme",
   "devDependencies": {
     "cypress": "^2.1.0"
+  },
+  "dependencies": {
+    "elm": "^0.18.0",
+    "http-server": "^0.11.1"
```
Comment (nelsonic) [inline package.json]: 👍

**Pull request merged**